applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py

```diff
@@ -216,8 +224,25 @@ def AdvanceInTime(self, current_time):
     def ComputeDeltaTime(self):
         return self.settings["time_stepping"]["time_step"].GetDouble()
 
+    def SearchElement(self):
+        self.solver.SearchElement(self.grid_model_part, self.material_model_part, self.max_number_of_search_results, self.searching_tolerance)
+
+    def InitializeSolutionStep(self):
+        self.SearchElement()
+        self.solver.Initialize()
```

Comment:
I put the Initialize() as a separate function and call it in here @philbucher. This works.